kedro/extras/datasets/pickle/pickle_dataset.py

```diff
@@ -95,14 +80,8 @@ class PickleDataSet(AbstractVersionedDataSet):
 
     DEFAULT_LOAD_ARGS = {}  # type: Dict[str, Any]
     DEFAULT_SAVE_ARGS = {}  # type: Dict[str, Any]
-    BACKENDS = {
```

Comment:
I think we may want to keep reference to the libraires we've tested despite allowing for an arbitrary module name? WDYT

Reply:
Agreed, I think it would be useful to add a note on which backends we know to work and which don't (e.g. torch).

Reply:
Added references to the ones that work in the docstring in https://github.com/quantumblacklabs/kedro/pull/931/commits/ed57e7c0cc5927a0b857594f0145c09dbf0ab762